docs/examples/testing/integration_test.py

```diff
@@ -19,25 +21,26 @@ def remote_method(self, value):
 
 
 class ServiceY(object):
+    """ Service under test
+    """
     name = "service_y"
 
     @rpc
     def append_identifier(self, value):
         return "{}-y".format(value)
 
-# =============================================================================
-# Begin test
-# =============================================================================
 
-def test_service_integration():
+def test_service_x_y_integration():
+
+    # run services in the normal manner
     config = {'AMQP_URI': 'amqp://guest:guest@localhost:5672/'}
     runner = ServiceRunner(config)
     runner.add_service(ServiceX)
     runner.add_service(ServiceY)
     runner.start()
 
+    # manually fire the "remote_method" entrypoint and verify response
     container = get_container(runner, ServiceX)
```

Comment:
I found it a bit confusing that in every example in the integration test section, we are obtaining the container in a different manner (`get_container` here, `container_factory` on restrict_entrypoints, `ServiceContainer` on replace_injections). 

Also, as a matter of fact, nowhere else in the docs is mentioned what a container is, which adds to the confusion.